BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Help/Views/SkontoHelpItemView.swift

```diff
@@ -0,0 +1,102 @@
+        titleLabel.accessibilityValue = content.title
+
+        addSubview(iconImageView)
+        addSubview(titleLabel)
+        if !hideDivider {
+            addDividerView()
+        }
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            iconImageView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor,
+                                               constant: Constants.verticalPadding),
+            iconImageView.leadingAnchor.constraint(equalTo: leadingAnchor,
+                                                   constant: Constants.horizontalPadding),
+            iconImageView.heightAnchor.constraint(equalToConstant: Constants.iconSize.height),
+            iconImageView.widthAnchor.constraint(equalToConstant: Constants.iconSize.width),
+            iconImageView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor,
+                                                  constant: -Constants.verticalPadding),
+
+            titleLabel.topAnchor.constraint(greaterThanOrEqualTo: topAnchor,
+                                            constant: Constants.verticalPadding),
+            titleLabel.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor,
+                                               constant: -Constants.verticalPadding),
+            titleLabel.leadingAnchor.constraint(equalTo: iconImageView.trailingAnchor,
+                                                constant: Constants.horizontalPadding),
+            titleLabel.trailingAnchor.constraint(equalTo: trailingAnchor,
+                                                 constant: -Constants.horizontalPadding),
+            titleLabel.centerYAnchor.constraint(equalTo: iconImageView.centerYAnchor)
+        ])
+    }
+
+    private func addDividerView() {
+        let separatorView = UIView()
+        separatorView.translatesAutoresizingMaskIntoConstraints = false
+        separatorView.backgroundColor = .giniColorScheme().bg.divider.uiColor()
+        addSubview(separatorView)
+        NSLayoutConstraint.activate([
+            separatorView.bottomAnchor.constraint(equalTo: bottomAnchor),
+            separatorView.heightAnchor.constraint(equalToConstant: Constants.separatorViewHeight),
+            separatorView.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+            separatorView.trailingAnchor.constraint(equalTo: trailingAnchor,
+                                                constant: -Constants.horizontalPadding)
+        ])
+    }
+}
+
+private extension SkontoHelpItemView {
+    enum Constants {
+        static let horizontalPadding: CGFloat = 16
+        static let verticalPadding: CGFloat = 10
+        static let iconSize = CGSize(width: 48, height: 48)
+        static let separatorViewHeight: CGFloat = 1
+        static let separatorLeadingPadding: CGFloat = 80
+        static let separatorTrailingPadding: CGFloat = 16
+        static let labelNumberOfLines: Int = 0
+    }
+}
```

Comment:
[request] Please rename everything that contains separator into divider to keep one name for this. Thank you!